src/components/event-card/EventCard.vue

```diff
@@ -0,0 +1,208 @@
+                >
+                    <!-- @slot for the title of the event -->
+                    <slot name="event-card-header" />
+                </VsHeading>
+
+                <!-- @slot for the date of event -->
+                <p class="vs-event-card__header-date">
+                    <slot name="event-card-header-date" />
+                </p>
+            </div>
+            <div class="vs-event-card__content">
+                <!--@slot for event description-->
+                <div
+                    v-if="$slots['event-card-content-description']"
+                    class="vs-event-card__content-description"
+                    data-test="vs-event-card__content-description"
+                >
+                    <p>
+                        <slot name="event-card-content-description" />
+                    </p>
+                </div>
+                <div class="vs-event-card__content-details">
+                    <div
+                        v-if="$slots['event-card-content-details']"
+                        class="vs-event-card__content-details-event-details"
+                        data-test="vs-event-card__content-details-event-details"
+                    >
+                        <VsList unstyled>
+                            <!--@slot for event details-->
+                            <slot name="event-card-content-details" />
+                        </VsList>
+                    </div>
+                    <div
+                        class="vs-event-card__content-details-travel-trade-details"
+                        data-test="vs-event-card__content-details-travel-trade-details"
+                        v-if="$slots['event-card-content-ttdetails']"
+                    >
+                        <VsList unstyled>
+                            <!-- @slot optional for second column details for travel trade events-->
+                            <slot name="event-card-content-ttdetails" />
+                        </VsList>
+                    </div>
+                    <div
+                        class="vs-event-card__content-details-event-cta"
+                        data-test="vs-event-card__content-details-event-cta"
+                        v-if="$slots['event-card-cta']"
+                    >
+                        <!-- @slot for the cta button -->
+                        <slot name="event-card-cta" />
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import VsHeading from '@/components/heading/Heading.vue';
+import VsList from '@/components/list/List.vue';
+
+/**
+ * The Event card component displays details about training & events
+ * for the industry in the Business Support hub website
+ *
+ * @displayName Event Card
+ */
+export default {
+    name: 'VsEventCard',
+    status: 'prototype',
+    release: '0.0.1',
+    components: {
+        VsHeading,
+        VsList,
+    },
+    props: {
+        ctaLink: {
```

Comment:
All props should have a comment - https://visitscotland.atlassian.net/wiki/spaces/DE/pages/159088753/Vue#Writing-a-Vue-component